src/extensions/additional/GPT/IconRefuge/IconRefuge.tsx

```diff
@@ -20,7 +17,7 @@ import './IconRefuge.scss';
  * refugeWidth and refugeHeight
  */
 
-export const IconRefuge: FC<IconRefugeProps> = ({
+export const IconRefuge: React.FC<IconRefugeProps> = ({
```

Comment:
nit:

`import type {FC} from 'react';` It seems to be more in line with the style of all the other places. No?